lib/stages/ReadGain.cpp

```diff
@@ -0,0 +1,139 @@
+#include "ReadGain.hpp"
+
+#include "buffer.h"
+#include "bufferContainer.hpp"
+#include "chimeMetadata.h"
+#include "configUpdater.hpp"
+#include "errors.h"
+#include <utils/visUtil.hpp>
+#include <functional>
+
+using kotekan::bufferContainer;
+using kotekan::Config;
+using kotekan::configUpdater;
+using kotekan::Stage;
+
+using kotekan::connectionInstance;
+using kotekan::HTTP_RESPONSE;
+using kotekan::restServer;
+
+REGISTER_KOTEKAN_STAGE(ReadGain);
+
+ReadGain::ReadGain(Config& config, const std::string& unique_name,
+                           bufferContainer& buffer_container) :
+  Stage(config, unique_name, buffer_container, std::bind(&ReadGain::main_thread, this)) {
+  //Apply config.
+  _num_elements = config.get<uint32_t>(unique_name, "num_elements");
+  scaling = config.get_default<float>(unique_name, "frb_scaling", 1.0);
+  vector<float> dg = {0.0, 0.0}; // re,im
+  default_gains = config.get_default<std::vector<float>>(unique_name, "frb_missing_gains", dg);
+
+  metadata_buf = get_buffer("network_buf");
```

Comment:
Can we change this to `in_buf` just to be more standardized with other stages.  (I know we aren't very consistent with this.)